migration/migration/002-utils.sql

```diff
@@ -11,14 +11,24 @@ CREATE TABLE _prom_catalog.remote_commands(
 GRANT ALL ON SEQUENCE _prom_catalog.remote_commands_seq_seq TO current_user;
 
 CREATE OR REPLACE PROCEDURE _prom_catalog.execute_everywhere(command_key text, command TEXT, transactional BOOLEAN = true)
+    SET search_path = pg_catalog
 AS $func$
+DECLARE
+    _is_restore_in_progress boolean = false;
 BEGIN
     IF command_key IS NOT NULL THEN
        INSERT INTO _prom_catalog.remote_commands(key, command, transactional) VALUES(command_key, command, transactional)
        ON CONFLICT (key) DO UPDATE SET command = excluded.command, transactional = excluded.transactional;
     END IF;
 
     EXECUTE command;
+
+    -- do not call distributed_exec if we are in the middle of restoring from backup
+    _is_restore_in_progress = coalesce((SELECT setting = 'on' from pg_catalog.pg_settings where name = 'timescaledb.restoring'), false);
```

Comment:
actually this query appears for the second time, should we make a function?

Reply:
i made it a function in idempotent/001-base.sql however unfortunately, the migration scripts run BEFORE the idempotent scripts so we cannot guarantee that it will exist at this point in the code